client/src/components/ExpenseDialog/ExpenseDialog.tsx

```diff
@@ -86,100 +127,116 @@ const ExpenseDialog: React.FC<ExpenseDialogProps> = ({
-                  <Typography variant="body1">$</Typography>
-                </InputAdornment>
-              ),
-            }}
-            inputProps={{
-              style: { color: theme.palette.primary.contrastText },
-            }}
-            InputLabelProps={{
-              style: { color: theme.palette.primary.contrastText },
-            }}
-            style={{ width: 550 }}
-          />
-          <TextField
-            select
-            id="category"
-            label="Category"
-            variant="filled"
-            value={formik.values.category}
-            onChange={(event) => {
-              event.target.name = 'category';
-              formik.handleChange(event);
-            }}
-            onBlur={formik.handleBlur}
-            error={formik.touched.category && Boolean(formik.errors.category)}
-            helperText={
-              formik.touched.category && Boolean(formik.errors.category)
-                ? formik.errors.category
-                : ' '
-            }
-            inputProps={{
-              style: { color: theme.palette.primary.contrastText },
-            }}
-            InputLabelProps={{
-              style: { color: theme.palette.primary.contrastText },
-            }}
-            style={{ width: 550 }}
-          >
-            <MenuItem value="GROCERIES" key="GROCERIES" color="#FFFFFF">
-              GROCERIES
-            </MenuItem>
-            <MenuItem value="TRANSPORT" key="TRANSPORT">
-              TRANSPORT
-            </MenuItem>
-            <MenuItem value="ENTERTAINMENT" key="ENTERTAINMENT">
-              ENTERTAINMENT
-            </MenuItem>
-            <MenuItem value="HEALTH" key="HEALTH">
-              HEALTH
-            </MenuItem>
-            <MenuItem value="UTILITIES" key="UTILITIES">
-              UTILITIES
-            </MenuItem>
-            <MenuItem value="OTHER" key="OTHER">
-              OTHER
-            </MenuItem>
-          </TextField>
+          <Box sx={{ display: 'flex', flexDirection: 'row', gap: '10px' }}>
+            <TextField
+              id="title"
+              label="Title"
+              variant="filled"
+              value={formik.values.title}
+              onChange={formik.handleChange}
+              onBlur={formik.handleBlur}
+              error={formik.touched.title && Boolean(formik.errors.title)}
+              helperText={
+                formik.touched.title && Boolean(formik.errors.title)
+                  ? formik.errors.title
+                  : ' '
+              }
+              inputProps={{
+                style: {
+                  color: theme.palette.primary.contrastText,
+                },
+              }}
+              InputLabelProps={{
+                style: { color: theme.palette.primary.contrastText },
+              }}
+              style={{ width: 300 }}
+            />
+            <TextField
+              id="createdAt"
+              label="Date"
```

Comment:
I think it wouldn't hurt to add the format in the label just to make it super explicit like `Date (yyyy/mm/dd)`

Reply:
wouldn't that be redundant? users wont be able to enter a date in a different format, and all rows in the table will have that one format

Reply:
Didnt realize it gives an error when it has the incorrect date format so tis all gud
![image](https://github.com/BarbzCodez/Spendr/assets/93229091/36987ad3-2cb5-451f-9bf8-f8120cde6769)